specs/0112-new-contributor-guidelines.md

```diff
@@ -1,25 +1,36 @@
-- Feature Name: Guidelines for new contributors
+- Feature Name: Team guidelines for new contributors
 - Created: 2022-05-31
 - Associated Issue: #2305
 
 # Summary
 
-Provide new contributors with guidelines when making their first contribution.
+Provide new contributors with guidelines before making their first contribution to a team.
 
 # Motivation
 
 It has been suggested on various occasions and from various sources, including at the localizer workshops, that the onboarding experience for new contributors needs improvements.
 
-Pontoon homepage and tour lead users towards making their first contribution, but they don't get much guidance since that point nor are they connected with their team. Locale Managers also don't get any direct information when new contributors join their teams.
+Pontoon homepage and tour lead users towards making their first contribution, but don't provide any team-specific information.
+
+On top of that, new users don't get connected with their team, and team managers don't get any explicit information about new contributors joining their teams.
 
 # Feature explanation
 
-After the user makes her first suggestion in Pontoon, we send an in-app notification with the following goal:
+## Translate view tooltip
+
+Before users submits their first suggestion to any team, a tooltip appears near the text editor, with the following content:
```

Comment:
We need a tangible value proposition for that, because it's not trivial to implement editing ability. And even if we had it, judging by the Info tab edits we can expect the majority of locales will not use it.

Why can't we put the same information into the resources we link to, like GitHub does (Info tab, Style Guides)?

Reply:
Because, for one, I would like to display that message in my own language, and be able to try different variants if it doesn't work, or add/remove information.

GitHub doesn't have anything localized, and it doesn't deal with localization as a primary function, so I don't think we should use it as a reference besides UI/UX ideas.

If we can't have it localizable, then we should keep this info to a minimum, and just link to the team info panel. 
* We link to style guides, but very few locales have one, and it's not necessarily maintained (which is a different conversation…).
* We link to contributors, but I don't believe that's helpful for newcomers, as there's no way to only display the managers for that locale (lot of people listed, including admins).

Reply:
That sounds more like we should finally start [localizing](https://github.com/mozilla/pontoon/issues?q=is%3Aopen+is%3Aissue+label%3Al10n) Pontoon or at least the translate view, where most of the i18n work has already been done.

I agree about the issue with the Contributors tab. To include that link, we should fix #2157 first.

I would keep the style guides, though, but only where applicable, obviously. The concerns you raise are valid for the Info tab, too. And, we already link to them from the locale dashboard.

Reply:
> I would keep the style guides, though, but only where applicable, obviously. The concerns you raise are valid for the Info tab, too. And, we already link to them from the locale dashboard.

Good point on the team info. Maybe we should add information in English for all locales if it's empty 🤔 

I really think that we shouldn't link to the style guides, because IMO that was done in the wrong way and it might even contain misleading information :-(

Reply:
Should we remove the style guides links from the DB (i.e. dashboards), too?